src/main/java/edu/kit/compiler/JavaEasyCompiler.java

```diff
@@ -386,6 +387,7 @@ public static void main(String[] args) {
                     new ArithmeticReplacementOptimization(),
                     new LinearBlocksOptimization(),
                     new InliningOptimization(),
+                    new LoopUnrollingOptimization(),
```

Comment:
Is there a reason for the exact placement of the `LoopUnrollingOptimization` in the list of optimizations?

Reply:
For example, we don't have to unroll loop invariant code.

Reply:
I mainly considered the interactions between inlining and loop unrolling. Moving loop invariant code first should allow us to unroll loops more frequently, as it may decrease the size of loop bodies. I'll move the optimization accordingly.